### PR TITLE
OCPBUGS-42349 Known issue for customer managed keys on Azure

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OCPBUGS-42349

Link to docs preview:
https://82365--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-known-issues_release-notes

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@jinyunma PTAL at this known issue for 4.17, TY

**Comment by jinyunma** [comment]
@bscott-rh thanks for addressing this, lgtm.
also cc @sadasu to review

**Comment by sadasu** [comment]
@bscott-rh thanks. LGTM but this needs a rebase. Will take a look at again after rebase is completed.

**Comment by mburke5678** [comment]
@bscott-rh One nit. Otherwise LGTM

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1954,6 +1954,8 @@ controlPlane:
 +
 As a temporary workaround, the image registry should not be configured as private when using the Azure File CSI driver. This is a known issue and will be fixed in a future version of {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-42308[*OCPBUGS-42308*])
 
+* When installing a cluster on {azure-short}, the installation will fail if a customer-managed encryption key is specified. (link:https://issues.redhat.com/browse/OCPBUGS-42349[*OCPBUGS-42349*])
```
[the remark this answers is not in the document]
Comment (bscott-rh) [reply release_notes/ocp-4-17-release-notes.adoc:1957]: Fixed, thx

**Pull request merged**